sotodlib/coords/pmat.py

```diff
@@ -470,7 +480,8 @@ def _get_proj_threads(self, cuts=None):
             else:
                 tile_info = proj.get_active_tiles(self._get_asm())
             self.active_tiles = tile_info['active_tiles']
-            proj = self._get_proj()
+            self.hp_geom.nside_tile = proj.nside_tile # Update nside_tile if it was 'auto'
```

Comment:
This line is healpix specific, doesn't it break non-healpix projector?

Reply:
It did; I had neglected to test tiled rectpix projectors. Fixed now